code/modules/projectiles/guns/specialist.dm

```diff
@@ -697,14 +678,12 @@
 
 /obj/item/weapon/gun/launcher/m92/mini_grenade/attackby(obj/item/I, mob/user, params)
 	if(!istype(I, /obj/item/explosive/grenade))
-		return ..()
-
+		return
 	var/obj/item/explosive/grenade/G = I
-	if(! (G.underslug_launchable))
+	if(!(G.underslug_launchable))
```

Comment:
if you are making both grenade launchers a subtype of eachother, you should make an allowed_grenade list and delete the underslug_launchable var. Its not a terribly big deal here tho.